scripts/download/lima.mjs

```diff
@@ -10,7 +10,7 @@ const limaRepo = 'https://github.com/rancher-sandbox/lima-and-qemu';
 const limaTag = 'v1.23';
 
 const alpineLimaRepo = 'https://github.com/lima-vm/alpine-lima';
-const alpineLimaTag = 'v0.2.11';
+const alpineLimaTag = 'v0.2.12';
```

Comment:
FWICT, if you want nerdctl 0.20.0, you need version 0.2.13, not 0.2.12

Reply:
Thanks, fixed!